app/src/main/java/org/mozilla/rocket/shopping/search/ui/ShoppingSearchResultTabFragment.kt

```diff
@@ -250,6 +252,17 @@ class ShoppingSearchResultTabFragment : Fragment(), ContentTabViewContract, Back
         })
     }
 
+    private fun calculateScrollXForTab(position: Int, positionOffset: Float): Int {
```

Comment:
Reference the implementation from TabLayout:
https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/tabs/TabLayout.java#L1784